### DISCUSSION TITLE
更新日志 - 22.03.6 - 20240126

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1311] [Announcements]
# 内核
1. 修复安装openwrt官方仓库提供的部分网络相关的内核模块时内核崩溃的问题。受影响机型：x86，rk33xx，rpi4。 #1276
2. r8125 移除刷屏的 “rss get rxnfc” 日志 #1291

# 系统
1. CGroups V1 切换到 CGroups V2（一般只跟 Docker/LXC 有关） #1284

# 应用
* QuickStart：修复用户自定义路由表时网关判断错误问题（多WAN且覆盖非默认路由表） #1218

**Comment by haobinnan** [comment]
希望下个版本 这样设置下 uci set system.@system[-1].cronloglevel='9'
要不 每次执行cron都会输出一行 cron.err 信息，这个还不是错误。

**Reply by jjm2473** [reply]
后台的系统-日志里面就可以调整啊。你看到的错误日志是什么，命令的输出吗，还是只是提示执行了什么命令？如果是命令的输出，可以使用管道重定向到/dev/null

**Reply by haobinnan** [reply]
Fri Jan 26 16:41:00 2024 cron.err crond[6359]: USER root pid 15400 cmd /etc/config/ddns.sh
Fri Jan 26 16:42:00 2024 cron.err crond[6359]: USER root pid 16361 cmd /etc/config/ddns.sh

**Comment by haobinnan** [comment]
https://fw.koolcenter.com/iStoreOS/ 
这里还没有看到新包

**Comment by haobinnan** [comment]
启动阶段的一些错误 看是否有必要处理下呢？

Fri Jan 26 20:41:40 2024 daemon.err ModemManager[6477]: hotplug: parent device sysfspath not found
Fri Jan 26 20:41:40 2024 daemon.err insmod: module is already loaded - pppox
Fri Jan 26 20:41:40 2024 daemon.err insmod: module is already loaded - pppoe

Fri Jan 26 20:41:41 2024 daemon.err ModemManager[6477]: hotplug: parent device sysfspath not found
Fri Jan 26 20:41:41 2024 daemon.err smbd[9084]: [2024/01/26 20:41:41.104264,  0] ../../source3/smbd/server.c:1091(smbd_open_one_socket)
Fri Jan 26 20:41:41 2024 daemon.err smbd[9084]:   smbd_open_one_socket: open_socket_in failed: Address not available
Fri Jan 26 20:41:41 2024 daemon.err smbd[9084]: [2024/01/26 20:41:41.104931,  0] ../../source3/smbd/server.c:1091(smbd_open_one_socket)
Fri Jan 26 20:41:41 2024 daemon.err smbd[9084]:   smbd_open_one_socket: open_socket_in failed: Address not available

Fri Jan 26 20:41:42 2024 daemon.err ttyd[11181]: [2024/01/26 20:41:42:7274] N:  Using foreign event loop...

Fri Jan 26 20:41:44 2024 daemon.err ModemManager[12044]: hotplug: parent device sysfspath not found

Fri Jan 26 20:41:45 2024 daemon.err ModemManager[12738]: hotplug: parent device sysfspath not found
Fri Jan 26 20:41:45 2024 daemon.err nmbd[9085]: [2024/01/26 20:41:45.349962,  0] ../../source3/nmbd/nmbd.c:59(terminate)
Fri Jan 26 20:41:45 2024 daemon.err nmbd[9085]:   Got SIGTERM: going down...
Fri Jan 26 20:41:45 2024 daemon.err nmbd[12809]: [2024/01/26 20:41:45.521536,  0] ../../source3/nmbd/nmbd.c:901(main)
Fri Jan 26 20:41:45 2024 daemon.err nmbd[12809]:   nmbd version 4.18.8 started.
Fri Jan 26 20:41:45 2024 daemon.err nmbd[12809]:   Copyright Andrew Tridgell and the Samba Team 1992-2023
Fri Jan 26 20:41:45 2024 daemon.err smbd[12811]: [2024/01/26 20:41:45.523893,  0] ../../source3/smbd/server.c:1746(main)
Fri Jan 26 20:41:45 2024 daemon.err smbd[12811]:   smbd version 4.18.8 started.
Fri Jan 26 20:41:45 2024 daemon.err smbd[12811]:   Copyright Andrew Tridgell and the Samba Team 1992-2023

Fri Jan 26 20:41:45 2024 daemon.debug ModemManager[12962]: hotplug: event reported: action=add, name=erspan0, subsystem=net
Fri Jan 26 20:41:45 2024 daemon.err ModemManager[12962]: hotplug: parent device sysfspath not found
Fri Jan 26 20:41:45 2024 daemon.debug ModemManager[13021]: hotplug: event reported: action=add, name=ip6gre0, subsystem=net
Fri Jan 26 20:41:45 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:45.910994257Z" level=warning msg="grpc: addrConn.createTransport failed to connect to {unix:///var/run/docker/containerd/containerd.sock  <nil> 0 <nil>}. Err :connection error: desc = \"transport: error while dialing: dial unix:///var/run/docker/containerd/containerd.sock: timeout\". Reconnecting..." module=grpc
Fri Jan 26 20:41:45 2024 daemon.err ModemManager[13021]: hotplug: parent device sysfspath not found

Fri Jan 26 20:41:46 2024 daemon.err ModemManager[13365]: hotplug: parent device sysfspath not found

Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47Z" level=warning msg="containerd config version `1` has been deprecated and will be removed in containerd v2.0, please switch to version `2`, see https://github.com/containerd/containerd/blob/main/docs/PLUGINS.md#version-header"
Fri Jan 26 20:41:47 2024 daemon.err modprobe: failed to find a module named aufs
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.624351471Z" level=warning msg="failed to load plugin io.containerd.snapshotter.v1.devmapper" error="devmapper not configured"
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.628922763Z" level=warning msg="could not use snapshotter devmapper in metadata plugin" error="devmapper not configured"
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.641068640Z" level=error msg="failed to initialize a tracing processor \"otlp\"" error="no OpenTelemetry endpoint: skip plugin"
Fri Jan 26 20:41:47 2024 kern.err kernel: [   30.999313] overlayfs: upperdir is in-use as upperdir/workdir of another mount, mount with '-o index=off' to override exclusive upperdir protection.
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.706237188Z" level=error msg="failed to mount overlay: device or resource busy" storage-driver=overlay2
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.706730688Z" level=error msg="exec: \"fuse-overlayfs\": executable file not found in $PATH" storage-driver=fuse-overlayfs
Fri Jan 26 20:41:47 2024 daemon.err modprobe: failed to find a module named aufs
Fri Jan 26 20:41:47 2024 daemon.err dockerd[9938]: time="2024-01-26T12:41:47.743504608Z" level=error msg="AUFS was not found in /proc/filesystems" storage-driver=aufs
Fri Jan 26 20:41:47 2024 kern.err kernel: [   31.039749] overlayfs: upperdir is in-use as upperdir/workdir of another mount, mount with '-o index=off' to override exclusive upperdir protection.

Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]: [2024/01/26 20:42:10.029657,  0] ../../source3/nmbd/nmbd_become_lmb.c:398(become_local_master_stage2)
Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]:   *****
Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]:
Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]:   Samba name server ISTOREOS is now a local master browser for workgroup OPENWRT on subnet 192.168.1.1
Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]:
Fri Jan 26 20:42:10 2024 daemon.err nmbd[12809]:   *****

Fri Jan 26 20:47:07 2024 daemon.err uhttpd[6218]: lsblk: /dev/mmcblk12: not a block device

Fri Jan 26 21:48:13 2024 daemon.err uhttpd[6218]: lsblk: /dev/mmcblk12: not a block device
Fri Jan 26 21:48:57 2024 daemon.err uhttpd[6218]: Not sandbox environment
Fri Jan 26 21:51:26 2024 daemon.err uhttpd[6218]: find: /tmp/appfilter/: No such file or directory


-------------------

设备型号
FastRhino R66s
固件版本
iStoreOS 22.03.6 2024012613

**Reply by jjm2473** [reply]
麻烦找openwrt解决，开机日志你都还介意。很多err只是警告而已。要么你就把有错误日志的插件都关掉，就不会有日志了。

**Comment by xixiluhui520** [comment]
老大救命，现在软路由突然不分配dhcp了，回复备份或者刷上一版系统都不能解决
![Uploading QQ浏览器截图20240204200740.jpg…]()

**Reply by xixiluhui520** [reply]
在线等，老大我现在真的没有办法了，快郁闷死了哦

**Reply by xixiluhui520** [reply]
![image](https://github.com/istoreos/istoreos/assets/139526512/0baac48c-ccca-4986-8601-59d78933d099)

**Reply by xixiluhui520** [reply]
![image](https://github.com/istoreos/istoreos/assets/139526512/6e929500-0a97-4e37-bb8b-25ae23821bca)

**Reply by q405812306** [reply]
![Screenshot 2024-02-07 062953](https://github.com/istoreos/istoreos/assets/34687385/204268ab-5e26-449f-b744-e43eb74ffd0f)
你这个点一下，重启就行了

**Comment by q405812306** [comment]
大佬，我发现个问题。（个人情况，不知道其他人是不是也是这样）
内网ipv4，公网ipv6，易有云能走ipv6直连，速度可以跑满上行带宽
在使用《系统便利工具》关闭ipv6后，易有云只能走中转服务器，速度相当的慢

问题：然后如果想恢复ipv6的话，《系统便利工具》里的 开启ipv6貌似是没有用的

![Screenshot 2024-02-07 063404](https://github.com/istoreos/istoreos/assets/34687385/2ef493de-4184-4cc2-ad8c-93d5940ecc6f)

**Reply by jjm2473** [reply]
那就在开启ipv6以后，修改lan接口的dhcp服务，把ipv6的服务都打开。

**Comment by gp1127** [comment]
大佬我是个新手，我装istoreos-22.03.5-2023121510-x86-64这个版本网卡显示千兆
![1111](https://github.com/istoreos/istoreos/assets/157882431/e918789f-b7f2-48a8-8686-8ccf13ee899d)
升级到最新的istoreos-22.03.6-2024012613-x86-64版本网卡显示是百兆
![222](https://github.com/istoreos/istoreos/assets/157882431/ae18b9e0-c54e-400b-8694-b2ff7d13a21a)

**Reply by jjm2473** [reply]
一般都是网线不好，有些版本的驱动要求比较严格。如果是realtek的网卡，在杂项设置-驱动里面选择驱动试试

**Reply by gp1127** [reply]
请问下。我这是要打哪个驱动
![3333](https://github.com/istoreos/istoreos/assets/157882431/f4a31845-bbeb-4927-9d6a-7075c9254d17)

**Reply by jjm2473** [reply]
"Realtek r8168"

**Reply by gp1127** [reply]
感谢开发者耐心解答~~~小白感谢万分~~祝：开发者新的一年万事如意，事业蒸蒸日上！！
